### ISSUE TITLE
Remake the DataModifier class.

### ISSUE THREAD

**Issue by Godsend72** [svarnypetr/PChackaton2016#20]
The class DataModifier needs a remake so that it imports and saves the data only once.
Other functions should just reads the imported data. 
Also the array column names should be changed with numpy, and not take the first row.
Docstrings for the functions should also be done.

**Comment by svarnypetr** [comment]
Fixed or just wontfix?

**Comment by Godsend72** [comment]
HI :D Well I ran into some problems when i wanted to set all arrays to the numpy ones. On each append it would be needed to provide some extra details. I don't quite remember as I gave up on that feature a few months ago. The rest is fixed.

**Comment by svarnypetr** [comment]
Hi, better to comment so that one knows what happened to the feature or why it was abbandoned.
